test/e2e/tests/get-protocolp-parameters.e2e.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, expect, test } from 'vitest';
+import { getFixtures } from '../utils/fixtures-loader.js';
+import { getWebSocketClient } from '../utils/setup-websocket-client.js';
+
+const fixtures = await getFixtures('get-protocol-parameters');
+
+describe('get-protocol-parameters', () => {
+  for (const fixture of fixtures) {
+    test(fixture.testName, async () => {
+      const ws = getWebSocketClient();
+      const { cost_models, ...data } = await ws.sendAndWait('GET_PROTOCOL_PARAMETERS');
+      expect(data).toMatchObject(fixture.result);
```

Comment:
```suggestion
      expect(data).toStrictEqual(fixture.result);
```
I prefer using `toStrictEqual` in this case. The fixture could be missing some data (as it is now missing `cost_models`) which could go unnoticed with a less strict assertion.